contracts/kill_switch/IKillSwitch.sol

```diff
@@ -0,0 +1,10 @@
+pragma solidity 0.4.24;
+
+import "./IIssuesRegistry.sol";
+
+
+contract IKillSwitch {
+    function initialize(IIssuesRegistry _issuesRegistry) external;
```

Comment:
Both here and `IIssuesRegistry`, I'd prefer removing `initialize()` from the interface.

It feels like an abstraction leak to have this here; it should only be the initializer / installer of these two apps that needs to know about its initialization protocol (another good reason to move the KillSwitch's initialization into the `DAOFactory`, as that's where both the ACL and KillSwitch are "real" rather than at the Kernel, where they're abstract).